Workspace/src/arm_interface/src/arm_motor_commands_node.cpp

```diff
@@ -0,0 +1,108 @@
+#include <ros/ros.h>
+#include <std_msgs/String.h> //includes string library 
+#include <stdbool.h>
+#include <list>
+#include <vector>
+
+struct msg_from_topics {
+	bool ik_status; 
+	std::list<float> data_points; 
+};
+
+class messageReceivedFromGUIWrapper {
```

Comment:
this isnt how you make custom ros messages. Please look at a ros tutorial on custom messages